pipeline-steps/terraform-plan-apply.yaml

```diff
@@ -0,0 +1,129 @@
+parameters:
```

Comment:
do we need this if we have a template for plan and a template for appy?

Reply:
both templates accept the same inputs, so makes sense put them into one, so we are not repeating the code in azure-pipelines.infra-core/wowza.yaml